test/read.js

```diff
@@ -52,7 +167,7 @@ describe("Testing jsonapi-client", function() {
       var somePhoto = person.photos[0];
       assert.throws(function() {
         somePhoto.relationships("photographer").add("foobar");
-      }, /Expected Resource, got String/);
+      }, /Expected Resource, got /);
```

Comment:
Believe it or not, IE9 and IE11 think `typeof "foobar" === undefined`, I've had to loosen this regex to enable the test suite to pass.